docs/content/docs/2.getting-started/2.installation.md

```diff
@@ -42,15 +78,26 @@ The `fpm-apache` variation is meant for users who want to run something like Wor
 [Learn more about using Docker with WordPress →](/docs/guide/using-wordpress-with-docker)
 
 ### FPM-NGINX
-The `fpm-nginx` variation is great for people who want to run Laravel applications or similar. This allows you to serve static content quickly with NGINX but also pass PHP requests to PHP-FPM. Similar to PHP-Apache, there are two processes required to run this variation. We use S6 Overlay to ensure the container health is accurate.
+Over the last 15+ years `fpm-nginx` has been "the way" to run PHP web applications, including Laravel. Since around 2010, it's been adopted and trusted by some of the best PHP system administrators out there. As of 2025, we're starting to see other options like FrankenPHP push the industry forward, but `fpm-nginx` is very stable and still widely used today.
+
+The only caveat with running FPM + NGINX together with Docker is it requires two processes (which can lead to strange container behaviors). Thankfully, we took this all in mind with our design by using 's been trusted and deployed millions of times and there's a large adoption This allows you to serve static content quickly with NGINX but also pass PHP requests to PHP-FPM. Similar to PHP-Apache, there are two processes required to run this variation. We use S6 Overlay to ensure the container health is accurate.
```

Comment:
This sentence doesn't read quite right.

> design by using 's been trusted

Reply:
Thanks! TONNNS of updates coming to the docs soon. Stay tuned 🤓